mantle/platform/api/aws/images.go

```diff
@@ -460,6 +460,24 @@ func (a *API) createImage(params *ec2.RegisterImageInput) (string, error) {
 	return imageID, nil
 }
 
+func (a *API) GrantVolumePermission(snapshotID string, userIDs []string) error {
```

Comment:
@LorbusChris It'd be really helpful if you could add a godoc for this function.

Reply:
added a comment, do you think it's sufficient?